### PR TITLE
Unocoin fixes

### PR DESCRIPTION
**Pull request by prwelber**
copy + style fixes

### CODE REVIEW

File: assets/js/components/exchange/exchange-verify.component.js [hunk 1]
```diff
@@ -67,6 +67,8 @@ function ExchangeVerifyController (Env, $scope, bcPhoneNumber, QA, unocoin, stat
   this.name = exchange.constructor.name.toLowerCase();
   this.showField = (field) => this.fields.indexOf(field) > -1;
 
+  this.capFirstLetter = (name) => name.charAt(0).toUpperCase() + name.slice(1);
```
Comment (plondon) [inline assets/js/components/exchange/exchange-verify.component.js]: Can't we do this with CSS?

**Comment by plondon** [comment]
I don't think these things are blocking really, so can we move to 1.20?

File: assets/js/components/exchange/exchange-verify.component.js (hunk 1 above)
Comment (prwelber) [inline assets/js/components/exchange/exchange-verify.component.js]: Yeah, good idea.

**Comment by Solen-Win** [comment]
Allowed

**Pull request merged**